src/StatusQ/Layout/StatusAppNavBar.qml

```diff
@@ -162,6 +169,16 @@ Rectangle {
         model: navBarModel
     }
 
+    Item {
```

Comment:
Could you use a `Loader` instead?

Reply:
This is same technique as `navBarProfileButtonSlot` I guess that is more logical to keep same logic